python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -147,31 +170,31 @@ def __init__(self, *args, **kwargs):
         self.unitScale = 1000.0 if self.config.toMilli else 1.0
 
     def run(self, patchRefList, cosmos=None):
-        haveForced = True  # do forced datasets exits (may not for single band datasets)
+        haveForced = False  # do forced datasets exits (may not for single band datasets)
         dataset = "Coadd_forced_src"
-        patchRefExistsList = [patchRef for patchRef in patchRefList if
-                              patchRef.datasetExists(self.config.coaddName + dataset)]
-        if len(patchRefExistsList) == 0:
-            haveForced = False
+        # Explicit input file was checked in CoaddAnalysisRunner, so a check on datasetExists
+        # is sufficient here (modulo the case where a forced dataset exists higher up the parent
+        # tree than the specified input, but does not exist in the input directory as the former
```

Comment:
Does the CoaddAnalysisRunner change prevent parent deepCoadd_forced_src from being used?